### ISSUE TITLE
[Bug] ReBasedLinearAttention AssertionError

### ISSUE THREAD

**Issue by marccasals98** [fla-org/flash-linear-attention#654]
### Checklist

- [x] I have checked [FAQs](https://github.com/fla-org/flash-linear-attention/blob/main/FAQs.md) and existing issues for similar problems
- [x] Please report this bug in English to ensure wider understanding and support

## Issue Description

When using `ReBasedLinearAttention` from the FLA library with default parameters, an `AssertionError` occurs during the forward pass due to a shape mismatch in the layer normalization weights.

### Stack Trace

```python
AssertionError in layer_norm_fwd
assert weight.shape == (G * D,)
```

### Debug Output

```
weight.shape: torch.Size([16]), G: 1, D: 64
```

The assertion fails because `weight.shape = [16]` but the expected shape is `[64]`.

## Root Cause

The `feature_dim` parameter in `ReBasedLinearAttention` defaults to `16`, but the layer normalization in the feature map expects the weight dimension to match the head dimension (`D = hidden_size / num_heads`).

When initializing with:
- `hidden_size=1024`
- `num_heads=16`

The head dimension becomes `D = 1024 / 16 = 64`, but the feature map's layer norm weights are initialized with the default `feature_dim=16`, causing the mismatch.

### Steps to Reproduce the Bug

```python
import torch
from fla.layers import ReBasedLinearAttention

emb_in = 1024
heads = 4
head_dim = emb_in // heads  # 1024 // 16 = 64
layer = ReBasedLinearAttention(hidden_size=emb_in, num_heads=heads, feature_dim=head_dim)
x = torch.randn(2, 128, emb_in)
out = layer(x)
print(f"Success! Output shape: {out.shape}")
```

### Expected Behavior

There should not be any error. You can try yourself by setting `heads=4`.

### Environment Information

- PyTorch: 2.9.0
- flash-linear-attention==0.4.0
- Triton 3.4.0
- Python: 3.11  
- Device: NVIDIA H100
- Distributed: DDP

**Comment by sustcsonglin** [comment]
fixed in https://github.com/fla-org/flash-linear-attention/commit/2de145f11b1320dc3e21385edbeeb9cb3fe07752
